app/decorators/course_decorator.rb

```diff
@@ -214,6 +214,10 @@ def selectable_master_subjects
     meta["edit_options"]["subjects"].map { |subject| [subject["attributes"]["subject_name"], subject["id"]] }
   end
 
+  def selectable_subordinate_subjects
+    selectable_master_subjects + [%w[None None]]
```

Comment:
Noob question: what is the purpose of this? => `[%w[None None]]`

Reply:
Guessing, but ... empty selection in the `select`?

Reply:
This is no longer needed and I need to remove it